crates/db/src/database.rs

```diff
@@ -1378,7 +1378,12 @@ impl DbProvider for Database {
                     let mut ft = Vec::new();
                     for dep in ix {
                         ft.push(CrateRegistryDep::from_index(
-                            get_desc_for_crate_dep(&self.db_con, &dep.name, &dep.registry).await?,
+                            get_desc_for_crate_dep(
+                                &self.db_con,
+                                &dep.name,
+                                dep.registry.as_deref(),
```

Comment:
`&` seems more concise than `as_deref()`

Reply:
See https://rust-lang.github.io/rust-clippy/master/index.html?groups=pedantic#ref_option -- and especially see the video linked from that page

Disclaimer:  I wrote that lint